package.json

```diff
@@ -37,13 +37,13 @@
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^1.7.3",
```

Comment:
@rwjblue upgrading this is possible, but i believe may be major breaking change, as I don't believe it is compatible with ember 2.16 (stuff we test via ember-try currently)